edx_proctoring/views.py

```diff
@@ -611,18 +611,16 @@ class StudentProctoredExamAttemptsByCourse(AuthenticatedAPIView):
     def get(self, request, course_id, search_by=None):  # pylint: disable=unused-argument
         """
         HTTP GET Handler. Returns the status of the exam attempt.
+        Course and Global staff can view both timed and proctored exam attempts.
         """
-        # course staff only views attempts of timed exams. edx staff can view both timed and proctored attempts.
-        time_exams_only = not request.user.is_staff
```

Comment:
Why can't we just simply update this line to change behavior instead of completely removing the functionality?

Reply:
Because we don't need this functionality anymore.